elytron/src/main/java/org/wildfly/extension/elytron/TrivialResourceDefinition.java

```diff
@@ -36,9 +37,11 @@ final class TrivialResourceDefinition extends SimpleResourceDefinition {
     private final Map<AttributeDefinition, OperationStepHandler> readOnlyAttributes;
     private final List<ResourceDefinition> children;
 
+    private final Stability stability;
```

Comment:
The SimpleResourceDefinition parent class already has a stability field.

Reply:
Fixed!